filter/example/src/App.js

```diff
@@ -1,7 +1,52 @@
-import React from "react";
+import React, { useState } from "react";
 import Filter from "filter";
 import FilterData from "./data.json";
-
+let data = {};
 export default function App() {
+  const airport = [
```

Comment:
Add proper comments, since it's managed by iCargo developer

Reply:
resolved